lib/rucio/core/replica.py

```diff
@@ -3136,7 +3138,7 @@ def get_replicas_state(scope=None, name=None, session=None):
 
 
 @read_session
-def get_suspicious_files(rse_expression, filter_=None, **kwargs):
+def get_suspicious_files(rse_expression, filter_=None, logger=logging.log, **kwargs):
```

Comment:
Any reason why `kwargs` is used here? We should really try to avoid this for core functions.

Reply:
The documentation needs a few changes, since the daemon has a different structure. I'll have to do that in the future.

Reply:
Ok, for me it's fine to do this later, please create a ticket for it though and create a PR later on 😄